src/components/FieldDateSelector/docs.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { Box, Button, Stack } from '@chakra-ui/react';
+import { Formiz } from '@formiz/core';
+import dayjs from 'dayjs';
+
+import { FieldDateSelector } from './index';
+
+export default {
+  title: 'Fields/FieldDateSelector',
+};
+
+export const Default = () => (
+  <Formiz onSubmit={console.log} autoForm>
+    <Stack spacing={4}>
+      <FieldDateSelector
+        name="demo"
+        label="Date"
+        helper="This is an helper"
+        required="Date is required"
+        defaultValue={dayjs()}
```

Comment:
Need check, but I don't think we want to pass DayJS values for a form field (see what we did on the FieldDayPicker componant, it's taking a Date object if I'm correct)